### PR TITLE
Fixes issue #425 where region-out is not triggered when seeking out of current region

### PR DESCRIPTION
**Pull request by jbeyer05**

### CODE REVIEW

File: plugin/wavesurfer.regions.js [hunk 1]
```diff
@@ -269,19 +269,22 @@ WaveSurfer.Region = {
     bindInOut: function () {
         var my = this;
 
+        my.firedIn = false;
+        my.firedOut = false;
+
         var onPlay = function () {
-            my.firedIn = false;
-            my.firedOut = false;
```
Comment (katspaugh) [inline plugin/wavesurfer.regions.js]: So `onPlay` is now empty? Why keep it?

Comment (jbeyer05) [inline plugin/wavesurfer.regions.js]: Would probably be cleaner with onPlay not there.  For those users who have custom code in onPlay, but are trying to stay current with the latest version of wavesurfer, it's easier for them if it's left there.
You want me to remove and re-submit the PR?

Comment (katspaugh) [inline plugin/wavesurfer.regions.js]: You don't have to re-submit the PR, just remove the function and push to the same branch, and this PR will be updated.

Comment (jbeyer05) [inline plugin/wavesurfer.regions.js]: done

**Comment by katspaugh** [comment]
Thanks!

**Pull request merged**